src/wp-includes/fonts.php

```diff
@@ -122,6 +122,17 @@ function wp_get_font_dir() {
 		'error'   => false,
 	);
 
+	wp_mkdir_p( $defaults['path'] );
+
+	if ( ! wp_is_writable( $defaults['path'] ) ) {
+		$defaults            = wp_upload_dir();
+		$defaults['path']    = path_join( $defaults['basedir'], 'fonts' );
+		$defaults['url']     = $defaults['baseurl'] . '/fonts';
+		$defaults['subdir']  = '';
+		$defaults['basedir'] = path_join( $defaults['basedir'], 'fonts' );
+		$defaults['baseurl'] = $defaults['baseurl'] . '/fonts';
```

Comment:
Nit: defaults can simply be redefined, ie `$defaults = array( /* etc */ );`

Reply:
done here: https://github.com/WordPress/wordpress-develop/pull/6259/commits/9b05b6a5dd1a8cb94acbd045995ca6e0412722de